lib/checks/tables/td-headers-attr.json

```
@@ -4,9 +4,13 @@
   "metadata": {
     "impact": "serious",
     "messages": {
-      "pass": "The headers attribute is exclusively used to refer to other cells in the table",
+      "pass": "The headers attribute is exclusively used to refer to other header cells in the table",
       "incomplete": "The headers attribute is empty",
-      "fail": "The headers attribute is not exclusively used to refer to other cells in the table"
+      "fail": {
+        "cell-header-not-in-table": "The headers attribute is not exclusively used to refer to other header cells in the table",
+        "cell-header-not-th": "The header attribute refers a data cell, this must be a header cell so that all screen readers announce it",
```

This should refer to "the headers attribute", not "the header attribute"

nit: I think it'd also be fine to be a bit more brief, maybe something like:

```suggestion
        "cell-header-not-th": "The headers attribute must refer to header cells, not data cells",
```